pkg/storer/reserve.go

```diff
@@ -7,7 +7,6 @@ package storer
 import (
```

Comment:
I would prefer if we have a separate file for the sampler code. I think this reserve file is already pretty big. This is more inline with this pkg.

Reply:
fixed